.github/workflows/no-important-files-changed.yml

```diff
@@ -0,0 +1,68 @@
+name: No important files changed
+
+on:
+  pull_request:
+    types: [opened]
```

Comment:
I think the idea was to only run the script when the PR is created because there is (currently) no deduplication logic. If it runs again on every PR change, I would expect multiple messages to be generated on a single PR (annoying). And I thought it's unlikely that a PR is created that doesn't touch the tests at all, but then later changes to tests are added. Not that it's impossible.

I think if we want to cover that situation as well, we need to make sure the message is only generated once.

Reply:
Ah you're right about the multiple messages, that's annoying. Let's keep it like this then. BTW, `edited` does not main it runs on every PR change, that's a separate push event.